bindings/nodejs/lib/types/block/core/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { plainToInstance } from 'class-transformer';
+import { BasicBlock } from './basic';
+import { BlockType } from './block';
+import { BlockWrapper } from './wrapper';
+
+export * from './wrapper';
+export * from './basic';
+export * from './block';
+
+export const BlockDiscriminator = {
```

Comment:
Feels like it should be in the block.ts file

Reply:
wil cause cyclic dep

Reply:
If you feel like it, I would add a small comment about that then. I see us asking why over and over again.

Reply:
Yes, no ?

Reply:
yes